Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,510 @@
+weight. Steel is very difficult and hardous to machine and is very heavy, although its strength is greater than aluminum. Plastics are tedious to machine as they tend to melt to the cutting
+blade and that casues the machining to stop frequently to clean off the blade. The propulsion has a very large pressure drop and this will cause a large temperature drop. A plastic structure will
+effected greatly by this and will harden to the point where fracture could occur very easily. The interfaces between the
+aluminum parts will also be included. This consists of the various t-slot connections and the bolts and washers that connect the aluminum sheets.
+
+There will also be an arduino connected to a computer that will use Simulink for data acquisition. The arduino will be a conduit to relay the data from the load cell
+to Simulink. There will also be an arduino to control the use of the solenoid. The solenoid will be activated from the Simulink program, which will coincide with
+the beginning of data acquisition.
+
+The system will also be displayed at Imagine RIT 2018. This will include a full poster and data representation.
+
+\subsubsection{Non-physical Deliverables}
+\label{subsubsec: non-physical deliverables}
+
+The non-physical deliverables of this project will be the project culmination report which will include a very detailed examination of the projects
+results. These results include the expected performance versus the test performance, the nozzle dimension polynomial approximation, the structure performance, and the
+propulsion system performance. The structure performance is a section that discusses the reasoning behind the initial design and any problems
+that were found during testing. The problems to be avoided will be, unecessary frictional loss due to bad connection points or instability of the test stand due to
+bad interfacing between structural components. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
+the mass flow under various ambient conditions and temperatures. The propulsion performance will mostly be measured through the data analysis. This specifically concerns the
+specific impulse. It will also concern the expansion of the flow, as the maximum thrust will occur when the exit pressure equals the ambient pressure and thus the shocks
+in the flow will expand optimally. This will be observed through schlieren lensing, as it has been performed before during nozzle testing. This was performed in the Spring 2017
+semester for flow visualization. The procedure was; turn the lights off, put the nozzle on the air hose, put a white piece of paper on a table, put it parallel to the table, initiate the cold burn, use a phone light
+to project the flow onto the white paper below it. Then distance between the nozzle and the light and the table were toggled until the best focus point was found. That is how the team
+was able to get the picture below.
```

Comment:
`That is how the team was able to get the picture below.`
Do not reference a figure's position on the page. Instead, reference its figure using `\autoref{fig:my-figure}` where `fig:my-figure` is the label attached to your figure.

I think you can omit that sentence entirely and add something like "shown in `\autoref{fig:nozzle-shocks}`" somewhere in the previous line.